build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'play'
+}
+
+repositories {
+    mavenLocal()
+    jcenter()
+}
+
+dependencies {
+    play 'com.sksamuel.scapegoat:scalac-scapegoat-plugin_2.11:1.0.0'
+    play 'com.typesafe.play:play-cache_2.11:2.5.18'         // cache
+    play 'com.typesafe.play:filters-helpers_2.11:2.5.18'    // filters
+    play 'com.typesafe.play:play-netty-server_2.11:2.5.18'
+    play 'com.typesafe.play:play-logback_2.11:2.5.18'
+    play 'com.typesafe.play:play-ws_2.11:2.5.18'            // ws
+    play 'com.typesafe.play:twirl-api_2.11:1.1.1'
+    play 'com.typesafe.scala-logging:scala-logging_2.11:3.5.0'
+    play 'io.lemonlabs:scala-uri_2.11:0.5.0'
+    play 'com.typesafe:config:1.3.1'
+    play 'io.swagger:swagger-play2_2.11:1.5.3'
+    play 'org.apache.hadoop:hadoop-common:2.6.0'
+    play 'org.apache.hbase:hbase-client:1.0.0'
+    play 'org.apache.hbase:hbase-common:1.0.0'
+    play 'org.webjars:swagger-ui:3.1.4'
+    play 'org.webjars:webjars-play_2.11:2.5.0-3'
+
+    playTest 'org.scalatestplus.play:scalatestplus-play_2.11:2.0.0'
+    playTest 'org.scalatest:scalatest_2.11:3.0.4'
+    playTest 'com.github.tomakehurst:wiremock:1.58'
+    playTest 'org.scalamock:scalamock_2.11:4.1.0'
+}
```

Comment:
what is the play / playTest prefix about?  (haven't seen that before)

Note that a number of the 'play' dependencies are not play related e.g. hbase.
Similarly, only the first 'playTest' dependency in fact relates to play.

Presumably playTest is automatically flagging the dependency as a test dependency?

Reply:
It's called a configuration in Gradle and allows the user to isolate dependencies. It's used to configure the classpath. Traditionally you might see `compile` or `test` in a Java/Groovy Gradle build. I'm not sure why they have used named configurations instead of the defaults, but I'm sure there's some detail in the implementation.